enterprise/views.py

```diff
@@ -637,7 +637,7 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             ),
             'course_modes': filter_audit_course_modes(enterprise_customer, course_modes),
             'course_effort': course_effort,
-            'course_full_description': clean_html_for_template_rendering(course_run['full_description']),
+            'course_full_description': clean_html_for_template_rendering(course_run['full_description'] or ''),
```

Comment:
@brittneyexline Why did we need this `clean_html_for_template_rendering` function?